orga/app/services/locale.js

```diff
@@ -4,10 +4,10 @@ import config from 'pix-orga/config/environment';
 export const FRENCH_INTERNATIONAL_LOCALE = 'fr';
 export const ENGLISH_INTERNATIONAL_LOCALE = 'en';
 export const FRENCH_FRANCE_LOCALE = 'fr-FR';
-export const DEFAULT_LOCALE = FRENCH_INTERNATIONAL_LOCALE;
+export const DEFAULT_LOCALE = 'fr';
```

Comment:
Voilà comment c'est fait dans #6182 qui est plus parlant :
```suggestion
export const DEFAULT_LOCALE = FRENCH_INTERNATIONAL_LOCALE;
```

ou alors encore ne plus utiliser `DEFAULT_LOCALE` dans ce fichier et mettre plutôt `FRENCH_INTERNATIONAL_LOCALE` directement ?

Reply:
Il y a un truc qui m'échappe sur la PR 6182, en quoi elle est plus parlante ? Je loupe un truc je crois :smile:
Je supprime le commit.
J'aurais préféré qu'on fasse une passe globale sur l'utilisation de ces constantes, on commençait à avoir un état à peu près stable, une constante = une valeur et là avant d'avoir fini, on dit que la défault est la french internationale locale sans avoir une vue d'ensemble.